tests/load-tests/pkg/journey/handle_integration_test_scenarios.go

```diff
@@ -1,17 +1,23 @@
 package journey
 
-import "fmt"
-import "strings"
-import "time"
-import "context"
-
-import logging "github.com/konflux-ci/e2e-tests/tests/load-tests/pkg/logging"
-
-import framework "github.com/konflux-ci/e2e-tests/pkg/framework"
-import util "github.com/devfile/library/v2/pkg/util"
-import utils "github.com/konflux-ci/e2e-tests/pkg/utils"
-import integrationApi "github.com/konflux-ci/integration-service/api/v1beta1"
-import types "k8s.io/apimachinery/pkg/types"
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	logging "github.com/konflux-ci/e2e-tests/tests/load-tests/pkg/logging"
+
+	framework "github.com/konflux-ci/e2e-tests/pkg/framework"
+
+	util "github.com/devfile/library/v2/pkg/util"
+
+	utils "github.com/konflux-ci/e2e-tests/pkg/utils"
+
+	integrationApi "github.com/konflux-ci/integration-service/api/v1beta2"
```

Comment:
I liked how this suite gave the API a generic name so we don't have to update its references all throughout the test suite.

Do you think it is worth naming the imports in other suites the same way? But this is totally optional, I'd approve the PR either way.

Reply:
I agree but it maybe a repo wide job for consistency and therefore a separate ticket.